internal/command/meta_backend.go

```diff
@@ -1513,7 +1513,8 @@ configuration or state have been made.
 const errBackendInitCloudMigration = `
 Reason: %s.
 
-Migrating to Terraform Cloud requires reinitialization, to discover which Terraform Cloud workspaces belong to this configuration and to optionally migrate existing state to the corresponding Terraform Cloud workspaces.
+Changes to the Terraform Cloud configuration block require reinitialization,
+to discover any changes to the available workspaces.
```

Comment:
Think hard-wrapping shouldn't be needed here, because this message ends up being the detail of a diagnostic and the diagnostic renderer has its own automatic wrapping logic (to help take into account the offset from the red bar Terraform renders along the left margin).

Reply:
Ah - we have an opportunity to consolidate error messages. Without the hard break I see this is the exact same message as `errBackendInitCloud`. I'll update the commit/PR.

Reply:
Updated